python/src/RandomVectorMetropolisHastings_doc.i.in

```diff
@@ -31,9 +31,9 @@ Examples
 
 Sample from a random variable:
 
->>> source = ot.RandomVector(ot.Normal())
+>>> randomVector = ot.RandomVector(ot.Normal())
 >>> initialState = [3.0]
->>> sampler = ot.RandomVectorMetropolisHastings(source, initialState)
+>>> sampler = ot.RandomVectorMetropolisHastings(randomVector, initialState)
 >>> sampler.setBurnIn(20)
 >>> sampler.setThinning(2)
```

Comment:
```suggestion
```
Makes no sense in this context: we are directly sampling from the random vector! No burn-in or thinning is required.